wrappers/services.go

```diff
@@ -209,6 +209,9 @@ func StartServices(apps []*snap.AppInfo, disabledSvcs []string, flags *StartServ
 			if e := systemSysd.Disable(toEnableSystem); e != nil {
 				inter.Notify(fmt.Sprintf("While trying to disable previously enabled services %q: %v", toEnableSystem, e))
 			}
+			if e := systemSysd.DaemonReload(); e != nil {
+				inter.Notify(fmt.Sprintf("While trying to do daemon-reload: %v", e))
+			}
 		}
 		if len(toEnableUser) > 0 {
 			if e := userSysd.Disable(toEnableUser); e != nil {
```

Comment:
shouldn't we add a userSysd.DaemoReload here?

Reply:
Actually double checked what happens in practice, just --global does not trigger a reload, but --global --user does apparently, but of the user instance. So for completeness I'll add a reload here too.

Reply:
Though the code intentionally panics when trying to call daemon-reload for user instances, which is intentional as there's no user instance associated with the snapd daemon, so we don't have to change anything here.

Reply:
but will the code behave correctly? from what you said the original code did a reload (but --global --user does apparently, but of the user instance) ?  do we need to keep reload variants of Enable/Disable and call them for user daemons?

Reply:
What I understand from the original code is that snapd does not have access to the session bus of the `root` user since it's running in a different context, so there is no user session daemon to ask for a reload.